src/lualib/NumberToFixed.ts

```diff
@@ -0,0 +1,13 @@
+/// https://www.ecma-international.org/ecma-262/10.0/index.html#sec-number.prototype.tofixed
+export function __TS__NumberToFixed(this: number, fractionDigits?: number): string {
+    if (Math.abs(this) >= 1e21 || this !== this) {
+        return this.toString();
+    }
+    const f = Math.floor(fractionDigits ?? 0);
+    if (f < 0 || f > 100) {
```

Comment:
Actually it may make sence to limit the fraction on ~80 to avoid "invalid format (width or precision too long)"

Reply:
I don't really care, even leaving out the check would be an option. Do what you think is best.